filebeat/crawler/prospector_log.go

```diff
@@ -38,7 +38,23 @@ func NewProspectorLog(p *Prospector) (*ProspectorLog, error) {
 
 func (p *ProspectorLog) Init() {
 	logp.Debug("prospector", "exclude_files: %s", p.config.ExcludeFiles)
-	p.scan()
+
+	logp.Info("Load previous states from registry into memory")
+
+	for path, fileinfo := range p.getFiles() {
```

Comment:
so we do not read files from registry that have not been found on startup? What happens if file re-appears?